src/main/scala/outwatch/Sink.scala

```diff
@@ -24,6 +24,8 @@ sealed trait Sink[-T] extends Any {
 
   private[outwatch] def observer: Subscriber[T]
 
+  def unsafeOnNext(value:T):Unit = { observer.onNext(value) }
```

Comment:
This should return `Future[Ack]`, so users can implement the back-pressure protocol.

Reply:
A more generic (and safer) version would be:
```scala
  def feed(values: T*): Future[Ack] = {
    Observer.feed(observer, values)
  }
```
This can feed multiple values into the observer while implementing back-pressure as well.

Reply:
Should I just take this function and rename it `unsafeOnNext`?

I'm a bit against varargs. What is the use-case for feeding multiple values in one run?

Reply:
No, I don't think this function should be named `unsafeOnNext`, I think `unsafeOnNext` should call `onNext` as in the PR, but have the correct return type (`Future[Ack]`).

I don't have a use case for it at the moment. I only suggested it because if somebody wanted to correctly feed multiple values with `unsafeOnNext`, he/she should handle the back-pressure protocol (only feed the next element when the future of the previous one completes with `Continue`) and `feed` does this.

Reply:
Ok. Let's tackle this once we have a use-case for the `feed` method. For now I'll just add `unsafeOnNext`.